guides/V1_share_space_template.md

```diff
@@ -0,0 +1,59 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'share-space-template'
+is: 'experimental'
+---
+
+# Share a Space Template
+
+A user can share one of their custom space templates with the _shareSpaceTemplate_ mutation.
+
+## Schema
+
+### Share Space Template Mutation
+At a high-level the mutation looks like this:
+
+```graphql
+MutationRoot {
+  shareSpaceTemplate(input: ShareSpaceTemplateInput!): ShareSpaceTemplateMutation!
+}
+
+type ShareSpaceTemplateInput {
+  id: ID!
+  audience: ShareAudience!
+}
+
+enum ShareAudience {
+  PUBLIC
+}
+```
+
+The input has two required arguments, id and audience. The id is the id of the template to be shared. The only audience currently available is PUBLIC, which means the template, once shared, is available to anyone that has permission to access publically shared templates. Once shared a template cannot be unshared.
+
+In order to share a template you should be the owner of the template and all required apps in the template should already be shared
```

Comment:
Clarify "owner" is creator? i.e., `"you should be the owner (i.e., creator) of the template"`